src/framework/components/scroll-view/component.js

```diff
@@ -482,6 +484,9 @@ class ScrollViewComponent extends Component {
 
                 this._setScrollFromContentPosition(position);
             }
+
+            this._velocity.x *= (1 - this.friction);
+            this._velocity.y *= (1 - this.friction);
```

Comment:
Note: moving the velocity update to *after* it's used this frame otherwise SCROLL_MODE_BOUNCE doesn't work properly when `friction = 1`.